src/import/load-costume.js

```diff
@@ -268,7 +268,14 @@ const loadCostumeFromAsset = function (costume, runtime, optVersion) {
                 return loadVector_(costume, runtime);
             });
     }
-    return loadBitmap_(costume, runtime, rotationCenter, optVersion);
+    return loadBitmap_(costume, runtime, rotationCenter, optVersion)
+        .catch(() => {
+            // Use default asset if original fails to load
+            costume.assetId = runtime.storage.defaultAssetId.ImageBitmap;
+            costume.asset = runtime.storage.get(costume.assetId);
+            costume.md5 = `${costume.assetId}.${AssetType.ImageBitmap.runtimeFormat}`;
+            return loadBitmap_(costume, runtime);
+        });
```

Comment:
Should there still be a console warning in here, so that it's easier for users/mods to investigate why a costume isn't loading?

Reply:
Probably. For consistency, however, that would require adding a warning to the vector loading code too, which makes it seem like a different change for a different PR.